### DISCUSSION TITLE
No new project template for Rider?

### DISCUSSION THREAD

**Discussion by resting** [intellij-rust/intellij-rust#10404] [Q&A]
Have installed the Rust plugin, but when I try to create a new solution, there's no Rust template to choose from.

Does the template not work for Rider and only for Idea Ultimate?

**Comment by mili-l** [comment]
Hello @resting, you can open existing projects and work with it in Rider, for now, there is no template to create a new one, but you can use [cargo new](https://doc.rust-lang.org/cargo/guide/creating-a-new-project.html) in [Terminal](https://www.jetbrains.com/help/rider/terminal-emulator.html) (View | Tool Windows | Terminal)
